RadixWallet/Clients/ProfileStore/ProfileStore.swift

```diff
@@ -587,6 +587,7 @@ extension ProfileStore {
 
 		let bdfs = try DeviceFactorSource.babylon(
 			mnemonicWithPassphrase: mnemonic,
+			isMain: true,
```

Comment:
this is inside `_newProfileAndBDFSMnemonic` function, so yes `Main`

Reply:
Why is the name underscored?

Reply:
to help visually disambiguate between public and private and static and non static. irrelevant to this PR :D feel free to rename them in ProfileStore in a seperate PR if you want :)